### PR TITLE
ignore the incoming message that was mistakenly included as pending a…

### PR DESCRIPTION
**Pull request by norkans7**
…nd acknoledget the command to prevent it to retry

### CODE REVIEW

File: temba/channels/views.py [hunk 1]
```diff
@@ -348,7 +348,10 @@ def sync(request, channel_id):
                         msg = Msg.objects.filter(pk=cmd['msg_id'], org=channel.org)
```
Comment (nicpottier) [inline temba/channels/views.py]: maybe change this to use .first() while we are in here.

File: temba/channels/views.py [hunk 2]
```diff
@@ -345,10 +345,12 @@ def sync(request, channel_id):
 
                     # catchall for commands that deal with a single message
                     if 'msg_id' in cmd:
-                        msg = Msg.objects.filter(pk=cmd['msg_id'], org=channel.org)
+                        msg = Msg.objects.filter(pk=cmd['msg_id'], org=channel.org).first()
                         if msg:
-                            msg = msg[0]
-                            handled = msg.update(cmd)
+                            if msg.direction == OUTGOING:
+                                handled = msg.update(cmd)
```
Comment (nicpottier) [inline temba/channels/views.py]: Oh actually this is going to fail now.. because `msg` is an object. Run those unit tests. :)

Comment (norkans7) [inline temba/channels/views.py]: yes that works `update` there is an instance method on Msg

Comment (nicpottier) [inline temba/channels/views.py]: oh, hah, sorry, you are absolutely right! 🤐

**Pull request merged**